keyboards/oktiboard/info.json

```diff
@@ -0,0 +1,15 @@
+﻿{
+    "keyboard_name": "oktiboard",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 4,
+    "height": 4,
+    "layouts": {
+      "LAYOUT_ortho_4x4": {
+        "layout": [{"x":0, "y":0}, {"x":1, "y":0}, {"x":2, "y":0}, {"x":3, "y":0}, {"x":0, "y":1}, {"x":1, "y":1}, {"x":2, "y":1}, {"x":3, "y":1}, {"x":0, "y":2}, {"x":1, "y":2}, {"x":2, "y":2}, {"x":3, "y":2}, {"x":0, "y":3}, {"x":1, "y":3}, {"x":2, "y":3}, {"x":3, "y":3}]
+      },
+      "LAYOUT_numpad_4x4": {
+        "layout": [{"x":0, "y":0}, {"x":1, "y":0}, {"x":2, "y":0}, {"x":3, "y":0, "h":2}, {"x":0, "y":1}, {"x":1, "y":1}, {"x":2, "y":1}, {"x":0, "y":2}, {"x":1, "y":2}, {"x":2, "y":2}, {"x":3, "y":2, "h":2}, {"x":0, "y":3, "w":2}, {"x":2, "y":3}]
```

Comment:
You need to either delete this block:

```suggestion
```

or add a `LAYOUT_numpad_4x4` macro to `oktiboard.h`.

Reply:
I think, prefer to delete this. 👍